### DISCUSSION TITLE
Crop images

### DISCUSSION THREAD

**Discussion by gillez** [photoprism/photoprism#3757] [General]
Is it possible to crop images in Photoprism, modifying the original or making a copy? If not, is there a plan to add this functionality?
I am moving to Photoprism from Google Photos after buying a Synology NAS. The one feature I really miss from Google Photos is the ability to crop photos. I feel this should be core functionality.

**Comment by gillez** [comment]
I've now created this as a new post in the Ideas board as I think that's a better place for it. Closing it here.

**Reply by graciousgrey** [reply]
Thank you very much for your feedback! 

The category of discussions is not so important. We are a small team and get a lot of questions every day. Therefore, we can not answer each one (in a timely manner) ourselves.

Cropping is not possible yet in PhotoPrism. 
However, you can use whichever tool you want to crop photos. If you choose a sequential name for the cropped image, PhotoPrism can stack it with the original during the next indexing.

https://docs.photoprism.app/user-guide/settings/library/#stacks

We might add cropping functionality in the future but at the moment we do have many to dos with a higher priority.

**Reply by gillez** [reply]
Thanks for the reply. Hopefully cropping will be implemented one day but until then I will use another tool for that task.

**Reply by chrisbansart** [reply]
I've just posted a proposition on that here : https://github.com/photoprism/photoprism/discussions/5360
